src/deb/control/postinst

```diff
@@ -1,2 +1,66 @@
-#!/bin/sh
-echo "executing post-install code..."
+#!/usr/bin/env bash
```

Comment:
Please specify with comments which parts of the postisnt are Tomcat boilerplate and which parts are unique to Rhino.